### PR TITLE
Add install_bcsymbolmap script

### PR DESCRIPTION
**Pull request by dacaiguoguogmail**
Unit test for https://github.com/CocoaPods/CocoaPods/pull/8470

### CODE REVIEW

File: install_vendored_dynamic_framework/before/Podfile [hunk 1]
```diff
@@ -31,4 +31,5 @@ target 'SampleApp' do
   use_frameworks!
   pod 'Realm', :podspec => 'https://gist.githubusercontent.com/segiddins/3a6d67837e0f84ea2f3f/raw/6867e04999d2e947fdefe7c9ff065c29815735cd/Realm.podspec'
   pod 'RealmSwift', :git => 'https://github.com/realm/realm-cocoa.git', :tag => 'v0.94.1'
+  pod 'BananaLib', :path => 'banana-lib'
```
Comment (dnkoutso) [inline install_vendored_dynamic_framework/before/Podfile]: Why is this added?

**Comment by dacaiguoguogmail** [comment]
To be continue.
